### ISSUE TITLE
Possible to have two directories in Radium for VSTi Instruments and VST Effects separated?

### ISSUE THREAD

**Issue by Lykdraft** [kmatheussen/radium#599]
Should speak for itself.

Usually plugins are in two separate locations in an application.

One only for the instruments and the other one only for the effects.

This would clean up the plugin folder a little bit more.

Best,

tL.

**Comment by onukore** [comment]
+1 and differente color object in mixer for fx an generator

**Comment by kmatheussen** [comment]
onukore: You can configure this in Preferences -> VST:
Lydkraft: I'll see if it makes sense to enable vst path configuration on OSX.

**Comment by kmatheussen** [comment]
Perhaps I misunderstood. On OSX, it seems like you are not usually requested to select where you want to install VST plugins. I guess you mean that Radium should sort all VST plugins automatically based in whether it is an instrument, and then group them into two different menues?

**Comment by Lykdraft** [comment]
That is correct. In most other applications you have (at least) two different menu-structures.
One for Instruments and one for Effects.
